### PR TITLE
RF | POM - EntityBoardDraftRecordTest> testCancelRecord()

### PR DESCRIPTION
**Pull request by MariaMoroz**
https://trello.com/c/rhVFwyII/1007-rf-pom-classname-testname

### CODE REVIEW

File: src/test/java/model/BoardEditPage.java [hunk 1]
```diff
@@ -113,4 +116,10 @@ public BoardPage clickSaveDraft(){
 
         return new BoardPage(getDriver());
     }
+
+    public BoardPage clickCancelDraft(){
```
Comment (MichaelMicrof) [inline src/test/java/model/BoardEditPage.java:120]: why is name of method clickCancelDRAFT?

**Comment by MariaMoroz** [comment]
издержки команды copy-paste)

сб, 19 июн. 2021 г. в 22:03, Mikhail Mir ***@***.***>:

File: src/test/java/EntityBoardDraftRecordTest.java [hunk 2]
```diff
@@ -129,4 +126,23 @@ public void testDeleteRecordFromRecycleBin() {
...
+
+        boardPage.clickNewButton()
+                .fillString(STRING_INPUT_PENDING)
+                .fillText(TEXT_VALUE_PENDING)
+                .fillInt(INT_VALUE_PENDING)
+                .fillDecimal(DECIMAL_VALUE_PENDING)
+                .clickCancelDraft();
+
+        String textCardBodyAfterCancel = boardPage.getTextCardBody();
+        Assert.assertEquals(textCardBodyAfterCancel, textCardBodyBeforeCancel);
```
Comment (MichaelMicrof) [inline src/test/java/EntityBoardDraftRecordTest.java:146]: Here we can use method isTableEmpty from BoardListPage

Comment (MariaMoroz) [reply src/test/java/EntityBoardDraftRecordTest.java:146]: No, we can't. In my Entity card body is not empty on BoardPage

File: src/test/java/model/BoardEditPage.java (hunk 1 above)
Comment (MariaMoroz) [reply src/test/java/model/BoardEditPage.java]: fixed

File: src/test/java/EntityBoardDraftRecordTest.java (hunk 2 above)
Comment (MariaMoroz) [reply src/test/java/EntityBoardDraftRecordTest.java:146]: No, we can't. In my Entity card body is not empty on BoardPage

**Review by MichaelMicrof — APPROVED** [review]

**Review by ValeriiSarukhanov — APPROVED** [review]

**Pull request merged**